reference/orders.v2.oas2.yml

```
@@ -2049,6 +2050,7 @@ components:
                   bin_picking_number: BIN
                   external_id: null
                   fulfillment_source: ''
+                  Brand: BigCommerce
```

```suggestion
                  brand: BigCommerce
```
Can you confirm if brand is with a lowercase b or uppercase B? I assume it is lowercase (brand).